src/sidebars/sidebar.html

```diff
@@ -0,0 +1,32 @@
+<div class="sidebars" ng-controller="SidebarCtrl as sidebarCtrl">
+    <div class="favourites {{sidebarCtrl.favouritesClass()}}" ng-controller="FavouritesCtrl as favouritesCtrl" ng-click="sidebarCtrl.favouritesClick()">
```

Comment:
Given the controller is declared in the directive does it need to be (re)declared here?